screenpipe-vision/src/ocr.swift

```diff
@@ -92,7 +92,12 @@ public func performOCR(imageData: UnsafePointer<UInt8>, length: Int, width: Int,
       }
     }
 
-    textRequest.recognitionLanguages = ["zh-Hans", "zh-Hant", "en-US"]
+    var recognitionLanguages = try? VNRecognizeTextRequest().supportedRecognitionLanguages()
+    if recognitionLanguages == nil {
+        recognitionLanguages = ["zh-Hans", "zh-Hant", "en-US"]
```

Comment:
shouldn't you give CLI languages here?

say i pick german it should be here?

Reply:
supported languages `["en-US", "fr-FR", "it-IT", "de-DE", "es-ES", "pt-BR", "zh-Hans", "zh-Hant", "yue-Hans", "yue-Hant", "ko-KR", "ja-JP", "ru-RU", "uk-UA", "th-TH", "vi-VT", "ar-SA", "ars-SA"]`

Reply:
I'll update the code to accept the languages selected in the CLI. I think it will create some confusion if the user selects a language that isn't supported and we default to English.